### PR TITLE
Ctauto 286 cttcm 2482

### PR DESCRIPTION
**Pull request by jrdavisllbean**
The code in this branch should automate all or most steps in CTTCM-2482.

**Changelog**
1) Added method to ProductPage named adqrioInformation() which returns a section class named ADQRIO. Most adq/rio elements are accessible via Selenium but a few are not. A combination JSoup/WebDriver would make testing this functionality easier. If we want to remove then no problem, I will delete.

2) Removed all hardcoded data from CTTCM_2482.java, replaced with checkoutUserInfoProperties properties.

3) Modified the ShoppingBag  getShoppingBagItemTotal() method to handle free monogram, engraving and to not include in total. Passing text to a BigDecimal throws a number format exception.

4) Removed call to the ProductPage selectRandomItem method in CTTCM_2482 (line 74). This was causing the test to fail in chrome.

5) Renamed PaymentOptions, PaymentOptionsPage.

6) Merged with the 10/10/16 12:00 pm EDT version of master.

### CODE REVIEW

**Comment by jrdavisllbean** [comment]
To test, please change class name in the TestSuite.xml to com.llbean.automation.web.tests.CTTCM_2498. I have confirmed the code works against QAE12, QAE13 and have not run against production.

**Comment by jrdavisllbean** [comment]
I just added LL Bean Visa Coupon to the test and the order summary class as well. Please run a git pull when you have a free moment.

File: src/main/java/com/llbean/automation/web/pages/desktopNTablet/sections/ADQRIO.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/**
+ * Created by jrdavis1 on 10/5/2016.
+ */
+public class ADQRIO extends BaseWeb {
+
+    private Document document;
+    private static final String IN_STOCK = "In Stock";
+
```
Comment (ecoronado89) [inline src/main/java/com/llbean/automation/web/pages/desktopNTablet/sections/ADQRIO.java]: Please add this elements to PageLocators class

**Comment by jrdavisllbean** [comment]
Hi @ecoronado89 , I have committed the following changes per your feedback. 

1) Move all hardcoded AADRIO selenium locators to the PageLocators interface.

2) To resolve the PASB -> ShoppingBag, PASB -> Product Page race condition, I added more robust waits to the AddedToYourShoppingBagModalPage clickCheckoutbtn and getContinuebtn methods. Eder my test should pass every run assuming the system is not slow. Please let me know if this is not the case.

Also should we add a method in BasePageObject like clickOnFirstVisibleElement(List<WebElement>)? If so I will add and replace in the AddedToYourShoppingBagModalPage clickCheckoutbtn and getContinuebtn methods. My change is messy and probably should be cleaned up.

**Pull request merged**